client/packages/coldchain/src/Monitoring/ListView/TemperatureBreach/TemperatureBreachList.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { FC } from 'react';
+import { useUrlQueryParams } from '@common/hooks';
+import { useTranslation } from '@common/intl';
+import {
+  CircleAlertIcon,
+  DataTable,
+  Formatter,
+  NothingHere,
+  TableProvider,
+  createTableStore,
+  useColumns,
+  useTheme,
+} from '@openmsupply-client/common';
+import {
+  TemperatureBreachFragment,
+  useTemperatureBreach,
+} from '../../api/TemperatureBreach';
+import { BreachTypeCell } from '../../../common';
+
+const ListView: FC = () => {
+  const { data, isLoading, isError } = useTemperatureBreach.document.list();
+  const {
+    updateSortQuery,
+    updatePaginationQuery,
+    // filter,
+    queryParams: { sortBy, page, first, offset },
+  } = useUrlQueryParams({ filterKey: 'startDatetime' });
+
+  const pagination = { page, first, offset };
+  const t = useTranslation('coldchain');
+  const theme = useTheme();
+
+  const columns = useColumns<TemperatureBreachFragment>(
+    [
+      {
+        key: 'acknowledgedIcon',
+        Cell: ({ rowData }) => {
+          return !rowData?.acknowledged ? (
+            <CircleAlertIcon
+              fill={theme.palette.error.main}
+              sx={{ color: '#fff' }}
```

Comment:
this is possibly not helpful. just wondered if we could remove a hard coded something.
```suggestion
              sx={{ color: 'background.white' }}
```